.gitignore

```diff
@@ -2,7 +2,7 @@
 *.exe
 *.o
 bongo
-bin/*
+build/*
```

Comment:
https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

:) Not necessary for modern meson versions.